src/builtin.jq

```diff
@@ -14,6 +14,9 @@ def _assign(paths; $value): reduce path(paths) as $p (.; setpath($p; $value));
 def _modify(paths; update): reduce path(paths) as $p (.; label $out | (setpath($p; getpath($p) | update) | ., break $out), delpaths([$p]));
 def map_values(f): .[] |= f;
 
+def base(a): [while(. > 0; (. / a) | floor) | . % a] | reverse;
```

Comment:
Use `$a` instead of `a` to use the argument as a value. Also, `f | . % a == f % a`.
```bash
❯ jq -nc 'def base(a): [while(. > 0; (. / a) | floor) % a] | reverse; 14 | base(2,3,4)'
[1,1,3,1,1,1,2,0,1,1,2,1,1,3,3,2,1,1,0,1,1,1,2,1,1,1,2,1,1,0,1,2,1,1,1,1,1,1,0,0,1,1,0,1,1,1,1,0]

❯ jq -nc 'def base($a): [while(. > 0; (. / $a) | floor) % $a] | reverse; 14 | base(2,3,4)'
[1,1,1,0]
[1,1,2]
[3,2]
```

Reply:
Fixed.